lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -715,4 +717,35 @@ main .attribute-access-type {
   font-family: var(--font-code);
 }
 
+@media (max-width: 480px) {
+  nav {
+    width: 100%;
+  }
+
+  main {
+    margin: 1em auto;
+    padding: 0 1em;
+    max-width: 100%;
+  }
+
+  #navigation-toggle {
+    top: 10px;
+    right: 10px;
+  }
+
+  #navigation-toggle[aria-expanded="true"] {
```

Comment:
We're setting `left: auto` for when `aria-expanded` is both true and false, can this be collapsed into a single rule?

Reply:
Updated 👍